src/pages/developers/standards/_meta.json

```diff
@@ -0,0 +1,10 @@
+{
+  "nft": {
+    "title": "Universal NFT",
+    "description": "Universal NFT."
+  },
+  "token": {
+    "title": "Universal Token",
+    "description": "Universal fungible tokens."
```

Comment:
The same could be said in this case, this probably works better as a brief description for the card:

> The Universal Token standard enables ERC-20 fungible tokens to be minted on any chain and seamlessly transferred between connected chains.